ext/standard/string.c

```diff
@@ -3863,6 +3866,38 @@ PHPAPI zend_string *php_addcslashes_str(const char *str, size_t len, const char
 }
 /* }}} */
 
+/* {{{ php_repr_str */
+PHPAPI zend_string *php_repr_str(const char *str, size_t len) {
```

Comment:
I think this function has the same output as `smart_str_append_escaped()` (minus the enclosing double quotes). Is there anything blocking us from using it?